webpack.config.js

```diff
@@ -284,8 +290,8 @@ module.exports = (env, argv) => {
       // https://webpack.js.org/plugins/split-chunks-plugin/#optimizationsplitchunks
       splitChunks: {
         chunks: 'async',
-        minSize: 30000,
-        maxSize: 0,
+        minSize: 30 * 1024,
```

Comment:
Why these values?

Reply:
30k is the default for minSize in webpack, I just rewrote it when I changed maxSize.
maxSize 0 was not working as a "decide what you want" value for webpack 5 and it complained it was (obviously) lower than minSize, if I recall well.
So I set it to the maximum chunk size as recommended which is around 250k (not a hard recommendation though, just an order of magnitude)